api/serializers.py

```diff
@@ -12,22 +12,46 @@
 class ProfileSerializer(serializers.ModelSerializer):
```

Comment:
Shouldn't this also be returning the image instance so it can be displayed? I know you mentioned that Django doesn't like loading static files... Not sure if this makes sense but could we treat the image like a video?

Reply:
yeah we might not have a choice if we can't get media files to be treated like static files without using django templates. I can make this change if everyone else agrees.

Reply:
Let's have this discussion in discord or in a meeting

Reply:
see the latest updates below